apps/prairielearn/src/lib/github.js

```diff
@@ -50,25 +50,30 @@ async function createRepoFromTemplateAsync(client, repo, template) {
 
   // The above call will complete before the repo itself is actually ready to use,
   // so poll for a bit until all the files are finally copied in.
-  let repo_up = false;
-  const poll_time_ms = 100;
-  while (!repo_up) {
+  //
+  // We'll try for up to 30 seconds, polling every second.
+  for (let i = 0; i < 30; i++) {
     try {
       // If the repo is not ready yet, this will fail with "repo is empty"
       await client.repos.getContent({
         owner: config.githubCourseOwner,
         repo: repo,
-        path: '/',
+        path: '',
```

Comment:
GitHub was 400ing when this was `/` 🤷 per their docs, omitting the path is equivalent to fetching the root of the repo, so this is the right thing to do.